hack/clear-route-status.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+clear_status ()
+{
+	MY_JSON_BLOB=$(curl -s -X GET http://localhost:8001/oapi/v1/namespaces/$1/routes/$2/)
+	if [ $(echo "$MY_JSON_BLOB" |jq '.kind') == '"Route"' ]; then
+		MODIFIED_JSON=$(echo "$MY_JSON_BLOB" | jq 'del(.status.ingress)')
+		curl -s -X PUT http://localhost:8001/oapi/v1/namespaces/$1/routes/$2/status --data-binary "$MODIFIED_JSON" -H "Content-Type: application/json" > /dev/null
+		echo "route status for route $2 in namespace $1 cleared"
+	else
+		echo "Route $2 not found in namespace $1"
+	fi
+}
+
+USAGE="Usage:
+./clear-route-status.sh [namespace] [route-name | ALL]
```

Comment:
Ah, I think you missed a slight nuance.  It's the _router_ name that we want to take here.

So if I deploy a router named 'bob' and then delete that router, I need to find all routes who have a status from router 'bob' and delete them.